exam001/Views/ViewController.swift

```diff
@@ -0,0 +1,482 @@
+        table.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            table.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            table.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            table.topAnchor.constraint(equalTo: submitButton.bottomAnchor, constant: 15),
+//            table.heightAnchor.constraint(equalToConstant: 500)
+            table.heightAnchor.constraint(greaterThanOrEqualToConstant: 400),
+            table.bottomAnchor.constraint(equalTo: stack.bottomAnchor, constant: -30)
+            ])
+    }
+    
+    private func setupFillerViewConstraints() {
+        fillerView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            fillerView.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            fillerView.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            fillerView.topAnchor.constraint(equalTo: table.bottomAnchor, constant: 15),
+            fillerView.bottomAnchor.constraint(equalTo: stack.bottomAnchor)
+            ])
+    }
+    
+    private func setupStackConstraints() {
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            stack.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 20),
+            stack.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -20),
+            stack.topAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.topAnchor, constant: 20),
+//            stack.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: 20)
+            stack.bottomAnchor.constraint(equalTo: containerView.bottomAnchor, constant: 20), // should be added on last item inside the container
+            ])
+    }
+    
+    private func setupSubstackConstraints() {
+        subStack.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            subStack.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            subStack.leftAnchor.constraint(equalTo: stack.leftAnchor)
+            ])
+    }
+    
+    // MARK: - tablview
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return fireBaseModel.getTableDatasize()
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = DevCell(style: UITableViewCell.CellStyle.value1, reuseIdentifier: "DevCell")
+        cell.cellIndex = indexPath.row
+        let singleCellData: NSDictionary = fireBaseModel.getTableData(indexPath.row)
+        let location = singleCellData[self.viewModel.fieldsArray[3]] as? String ?? ""
+        let stat = singleCellData["Status"] as? String ?? ""
+        cell.textLabel!.text = location
+        cell.detailTextLabel?.text = stat
+        cell.textLabel?.textColor = UIColor.orange
```

Comment:
`UIColor.orange `can be written as `.orange` instead
Always use the shortest compiled code